app/views/declaration/package_information.scala.html

```diff
@@ -23,99 +23,109 @@
-
-    @components.error_summary(form.errors)
-
-    @components.heading(
-        sectionHeader = messages("supplementary.items"),
-        title = messages("supplementary.packageInformation.title")
-    )
-
-    @helper.form(routes.PackageInformationController.submitForm(mode, itemId), 'autoComplete -> "off") {
-        @helper.CSRF.formField
-
-        @if(packages.nonEmpty) {
-            <table class="form-group">
-                <caption>@tableHeader</caption>
-                <thead>
-                    <tr>
-                        <th scope="col">@messages("supplementary.packageInformation.typesOfPackages")</th>
-                        <th scope="col">@messages("supplementary.packageInformation.numberOfPackages")</th>
-                        <th scope="col">@messages("supplementary.packageInformation.shippingMarks")</th>
-                        <td></td>
-                    </tr>
-                </thead>
-                <tbody>
-                @packages.zipWithIndex.map { case (item, index) =>
-                <tr>
-                    <th scope="row">@item.typesOfPackages</th>
-                    <td>@item.numberOfPackages</td>
-                    <td>@item.shippingMarks</td>
-                    <td>
-                        @components.buttons.remove_button(element = LabelledValue(item.toJson),
-                            hint=messages("supplementary.packageInformation.remove.hint",
-                                item.typesOfPackages,
-                                item.shippingMarks))
-                </button>
-                </td>
-                </tr>
-                }
-                </tbody>
-            </table>
-        }
-
-        @components.fields.field_autocomplete(
-            field = form("typesOfPackages"),
-            label = "supplementary.packageInformation.typesOfPackages",
-            labelClass = Some("form-label-bold"),
-            hintText = None,
-            emptySelectValue = messages("supplementary.packageInformation.typesOfPackages.empty"),
-            items = AutoCompleteItem.fromPackageType(PackageTypes.all),
-            'otherErrorFields -> Seq("typesOfPackages")
+    @formHelper(action = routes.PackageInformationController.submitForm(mode, itemId), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
+
+        @sectionHeader(messages("supplementary.items"))
+        @pageTitle(text = messages("declaration.packageInformation.title"), classes = "govuk-heading-xl")
+
+        @packageInfoTable
+
+        @components.fields.field_accessible_autocomplete(
+          field = form("typesOfPackages"),
+          label = messages("declaration.packageInformation.typesOfPackages"),
+          labelClass = Some("govuk-label--m"),
+          hintText = None,
+          emptySelectValue = messages("declaration.packageInformation.typesOfPackages.empty"),
+          items = AutoCompleteItem.fromPackageType(PackageTypes.all),
+          'otherErrorFields -> Seq("typesOfPackages")
```

Comment:
Why it's with `'` at the beginnig? I see that previously it was the same

Reply:
No idea. I allowed myself to copy the previously working solution as much as it is possible.